src/rules/noUnsafeAnyRule.ts

```diff
@@ -387,9 +388,28 @@ function isNodeAny(node: ts.Node, checker: ts.TypeChecker): boolean {
             return isAny(checker.getDeclaredTypeOfSymbol(symbol));
         }
     }
+
+    // Lowercase JSX elements are assumed to be allowed by design
+    if (isJsxNativeElement(node)) {
+        return false;
+    }
+
     return isAny(checker.getTypeAtLocation(node));
 }
 
+const jsxElementTypes = new Set<ts.SyntaxKind>([
+    ts.SyntaxKind.JsxClosingElement,
+    ts.SyntaxKind.JsxOpeningElement,
+    ts.SyntaxKind.JsxSelfClosingElement,
+]);
+
+function isJsxNativeElement(node: ts.Node): boolean {
+    return isIdentifier(node)
+        && node.parent !== undefined
+        && jsxElementTypes.has(node.parent.kind)
+        && isLowerCase(node.text[0]);
```

Comment:
instead of isLowerCase (which can allow false positives), it might be better to just get a set of all native jsx elements since they're exhaustive and well defined.

Reply:
> set of all native jsx elements

I think that's no longer the case: https://github.com/facebook/react/issues/2746

What kind of false positives should I be looking out for?

Off the top of my head, I'm thinking this might need to be changed to `isFirstLetterLowerCase` - trying to find a reference for what happens if you do something like `<myElement />`

Reply:
@JoshuaKGoldberg any update here?

Reply:
Thanks for the reminder - behaviorally, it looks like it's purely based on the first letter of the element name.

```jsx
<span>lowercase span</span>
<sPan>camelCase sPan</sPan>
```

becomes in create-react-app:

```html
<span>lowercase span</span>
<span>camelCase sPan</span>
```

Reply:
oh i see cool. Yeah I think it should be first letter lower case instead of lower case.